xbmc/cores/VideoPlayer/DVDCodecs/Video/VAAPI.cpp

```diff
@@ -123,14 +128,44 @@ bool CVAAPIContext::EnsureContext(CVAAPIContext **ctx, CDecoder *decoder)
   return true;
 }
 
+namespace
+{
+int find_open_render_node_fd()
```

Comment:
I'd say we start with ifdef until we have a better idea.
Are you aware about Kodi's release process? Currently we are in beta, close to RC stages. Means that new features are targeted to v18. So we are not in a hurry with this. Once v17 has branched, master should be open for business again.